tests/scripts/generate_psa_tests.py

```diff
@@ -265,12 +263,16 @@ def test_cases_for_key_type_not_supported(
 
```

Comment:
The dependency for DH key pairs is `PSA_WANT_KEY_TYPE_DH_KEY_PAIR`, so the tests are not executed. This needs to change to the new suffixes, like is already the case for ECC.